src/main/webapp/WEB-INF/messages.properties

```diff
@@ -657,6 +657,8 @@ month.12=December
 
 # Email
 #offlineEmailBody=Testing email body {0}
+offlineFailEmailBody=Your [hubName] download has failed. Please contact [support] by replying to this email and we will investigate the cause. <br><br>Your search URL was: <br> [searchUrl]<br><br>The reference to quote is:<br> [uniqueId] <br><br>Your successful downloads can be found at:<br> [myDownloadsUrl]
+offlineFailEmailBodyCSDM=Your [hubName] download has failed. You will not be able to import data for modelling. Please try again after some time or contact [support] by replying to this email and we will investigate the cause.
```

Comment:
Please include `[uniqueId]` in this error message so that the support team have a reference to start with.

Reply:
This has not been fixed